.travis.yml

```diff
@@ -5,8 +5,9 @@ python:
 
 # command to install dependencies
 install: 
-  - pip3 install -q -r requirements.txt
-  - pip3 install -q -r source/_scripts/_python/requirements.txt
+  - pip3 install -q virtualenv
+  - make venv
+  - source bin/activate/venv
```

Comment:
Note: Ln 10 command needs to be tested. In case the venv activation doesn't survive into the follow steps, like `scripts`, then we can replace these commands with only: `pip3 install -q -r requirements.txt`, as before.